repo-agent/pkg/codeserver/codeserver.go

```diff
@@ -0,0 +1,31 @@
+package codeserver
+
+import (
+	"fmt"
+	"log"
+	"os"
+	"os/exec"
+	"strings"
+)
+
+var execCommand = exec.Command
+
+func Start() (*exec.Cmd, error) {
+	log.Println("starting code-server")
```

Comment:
Reading the `GIT_HTML_URL` directly from an environment variable makes this package less reusable as it creates a hidden dependency. It's better to pass this value as a parameter to the `Start` function.